scalaz/src/main/scala/github4s/scalaz/Implicits.scala

```diff
@@ -0,0 +1,25 @@
+package github4s.scalaz
+
+import cats.MonadError
+import github4s.free.interpreters.Interpreters
+import scalaz.concurrent.Task
+
+object Implicits {
+
+  implicit val evalMonadError: MonadError[Task, Throwable] = new MonadError[Task, Throwable] {
+
+    override def pure[A](x: A): Task[A] = Task.now(x)
```

Comment:
I'd overwrite `pureEval` too for using `Task.now` when the value has been already evaluated, like [they do in Cats with Future](https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/std/future.scala#L16-L19). This avoids introducing an unneeded asynchronous boundary.